lci_strategic_plugin/src/main.cpp

```diff
@@ -89,6 +102,8 @@ int main(int argc, char** argv)
         tf_distance_pub.publish(tf_distance);
         earliest_et_pub.publish(earliest_et);
         et_pub.publish(scheduled_et);
+
+        lcip.onSpin(); 
```

Comment:
It would be better to create another timer to call this function at a configurable rate such that you can adjust the frequency of mobilityOperation publication without impacting the plugin discovery frequency or needing to rebuild. 

Also onSpin() could be renamed to something more meaningful since it only relates to mobilityOperation publication.